mock_discord.py

```
@@ -0,0 +1,189 @@
+    def __str__(self):
+        return self.name
+
+
+class MockChannel:
+    """Mock Discord Channel object"""
+    def __init__(self, channel_id: int = 987654321, name: str = "bot-devs"):
+        self.id = channel_id
+        self.name = name
+        self._messages: List[Dict[str, Any]] = []
+        
+    async def send(self, content: str, allowed_mentions=None):
```

Pylint (3.3.7)

[warning] Unused argument 'allowed_mentions'

(W0613)